u8timeseries/backtesting/backtesting.py

```diff
@@ -11,6 +11,10 @@
 
 from u8timeseries.utils import _build_tqdm_iterator
 from ..logging import raise_if_not, get_logger
+from ..utils.statistics import plot_acf
+import matplotlib.pyplot as plt
```

Comment:
Let's move external libs to the top

Reply:
In general how about python lib then external lib and at the end our modules

Reply:
done